io/EptReader.cpp

```diff
@@ -710,8 +712,10 @@ PointViewSet EptReader::run(PointViewPtr view)
 
             if (m_info->dataType() == EptInfo::DataType::Laszip)
                 startId = readLaszip(*view, key, nodeId);
-            else
+            else if (m_info->dataType() == EptInfo::DataType::Binary)
                 startId = readBinary(*view, key, nodeId);
+            else
```

Comment:
This is a bad default condition.

Reply:
ef58e49a6596edcbfe80b30169142b7b1f734a96